sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -394,6 +411,39 @@ def _upload_entity_df_and_get_entity_schema(
     return entity_schema
 
 
+def _get_entity_df_event_timestamp_range(
+    entity_df: Union[pd.DataFrame, str],
+    entity_df_event_timestamp_col: str,
+    client: Client,
+    table_name: str,
+) -> Tuple[datetime, datetime]:
+    if type(entity_df) is str:
```

Comment:
Is there a reason not to use `isinstance`?

Reply:
Good catch! Will change